lessons/06-html-layout/posts/post-2.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<head>
+  <Title>News Article 2</Title>
```

Comment:
Watch out for capitals, everything should be lowercase, e.g. `<title>`

Reply:
Thanks for mentioning ....Done